cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -565,29 +565,34 @@ func (impl *OSBuildJobImpl) Run(job worker.Job) error {
 				break
 			}
 
+			object := targetOptions.Object
+			if object == "" {
```

Comment:
Not a fan tbh. It means that both the Cloud API and the worker can generate names. I suggest doing it on just one layer.

Reply:
"doing it on just one layer" means in this case doing it either in two places (Weldr and Cloud API) or in worker.